src/tox/venv.py

```diff
@@ -387,7 +387,7 @@ def ensure_pip_os_environ_ok(self):
             os.environ.pop(key, None)
         if "PYTHONPATH" not in self.envconfig.passenv:
             # If PYTHONPATH not explicitly asked for, remove it.
-            if "PYTHONPATH" in os.environ:
+            if os.environ.get("PYTHONPATH"):
```

Comment:
note that this is a behavior change allowing the empty variable to pass trough

Reply:
Yeah I considered that (and popping it out) but couldn't think of a decent reason why it'd matter.

Happy to change if someone does though.

Reply:
Well, at least it will make the env larger, and therefore I think it should not be added in the first place - it is nicer to have less in `env` etc.

Reply:
let's err on the side of caution and remove it anyway.  can dedent line 395 and change it to `os.environ.pop('PYTHONPATH', None)`